### ISSUE TITLE
Crash when trying to create a world

### ISSUE THREAD

**Issue by Sunconure11** [Juuxel/WoodsAndMires#16]
https://pastebin.com/hdaPGHnh

Removing mods of potential culprits results in hanging.

[latest.log](https://github.com/Juuxel/WoodsAndMires/files/11306603/latest.log)

Of note is that super flats work fine.

**Comment by Sunconure11** [comment]
This is related to Rusticated.

https://github.com/cScotPlay/Rusticated/issues/13

@Juuxel

**Comment by Juuxel** [comment]
Hmm, Rusticated breaks since my pine forests are in both `is_taiga` and `is_forest`, so it adds the same feature twice.

https://github.com/cScotPlay/Rusticated/blob/33b0c806b93af964653c79985af3d602cff97e82/src/main/java/net/mcs3/rusticated/world/level/levelgen/generation/VegetationGeneration.java#L28-L29

**Comment by Sunconure11** [comment]
I'm going to see if the dev forgot about this.

**Comment by Sunconure11** [comment]
Yeah, I'll send over a PR to them. You said something about to OR the features, any examples of that here?

**Comment by Sunconure11** [comment]
I've also brought this up in the world gen mod dev discord @Juuxel

**Comment by Sunconure11** [comment]
I have sent over a PR to them that will fix it. They will need to merge it.
